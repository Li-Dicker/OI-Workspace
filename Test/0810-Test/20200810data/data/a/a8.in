10
198
42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 42134 
33780 43256 45019 36179 47219 16200 29374 44157 13445 28077 31407 18063 27750 40202 2850 8892 30569 35576 25289 24942 42038 25261 39413 49383 43315 49682 37258 702 43280 4958 14209 30302 33698 45994 4614 869 13072 48297 16638 42879 16299 32116 45772 47828 35832 15615 10377 48576 8532 33513 41014 35600 28189 28538 24331 16035 12769 11468 1017 5742 28948 28675 7891 22137 38026 33353 44538 34565 30448 18198 2631 40918 10326 20338 38372 9355 1013 15477 21917 14975 6852 20731 12863 2852 36674 43957 15476 47180 11765 17264 30514 46352 33882 31509 25879 22079 40737 6384 33278 12110 1628 18008 11314 2752 4697 30712 47663 6806 22127 48789 16997 37769 31246 48979 4117 22593 14565 20831 23390 48691 25338 1249 14900 7038 21724 45203 47308 15318 23412 9931 28581 6629 44736 43252 24275 33257 33724 1510 20053 25387 43488 12105 1986 48028 15690 38960 19824 23458 25866 20915 32056 48188 29195 696 18513 567 48537 2896 13427 3970 37956 22998 48935 5387 34098 15681 26195 37610 14510 11403 23648 28557 28042 31660 47113 9895 11088 34493 49798 10639 20767 13873 31667 1543 2133 35950 33621 9750 3127 38032 7860 7692 23551 4544 3505 34624 33022 30821 
197
82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 82976 
15005 45788 22828 9201 760 30983 5236 29895 10502 22082 40245 13083 47245 46815 41936 28402 17927 35532 27545 12379 40468 15508 16952 23042 22988 47438 41201 3001 28124 26287 291 22669 2946 20848 49267 23639 28444 36007 28738 17182 35962 21922 37230 606 23302 830 9947 1685 45559 27199 36996 27108 17510 9370 33497 38906 3739 38741 30274 30184 31634 31309 48115 33641 14734 32434 34891 27929 16386 26318 2379 26238 28881 1086 48825 13617 15799 2739 4718 21858 46673 35442 47846 38140 43416 8235 37572 38495 23289 25260 24457 11034 47774 30481 22488 4836 49280 13751 32419 34612 46253 11334 38162 27496 37809 22068 35318 42579 19873 42997 44123 12111 39806 14511 18323 15228 49127 44264 1506 25706 26396 26421 28500 2524 8542 10139 21635 39794 33748 21445 19706 36187 15212 31847 44998 31786 13018 30154 10087 24812 37150 43100 11544 3261 23175 26566 23866 34333 43560 29295 9741 15461 9217 35089 44062 31676 30427 45877 4799 38956 34285 11857 43236 17244 4929 37973 28652 9540 18595 1672 5141 36666 37932 16922 6206 43334 1200 33459 39934 38675 26307 36190 34608 44427 13396 37922 49361 46162 37496 24030 42253 15212 28057 29217 21563 21698 15691 
200
45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 45139 
17172 46697 48437 21218 13515 46894 23731 22323 39686 42686 14715 861 47981 25966 3654 21190 17562 45343 17416 47486 18593 20676 39662 21048 28486 14833 9714 31424 21334 45810 34134 14987 26120 45395 36283 4706 1307 45401 20245 6011 32470 13611 38087 6883 26458 3408 23690 46765 35217 2882 34175 5356 5195 33735 43725 23916 6617 2786 34344 30913 24285 32976 10730 48216 45207 7629 5787 41190 5470 36625 26230 35208 14166 43755 38164 34768 11961 46334 4641 41032 8468 2231 19919 17429 30511 48486 24615 41757 12171 14798 11295 44524 46264 48397 49023 4468 26010 13077 6436 24813 12420 41295 32249 38877 26665 7642 4793 15581 1574 31319 30501 2967 20938 17580 43813 44081 33352 49101 26989 26000 46757 10554 20064 11948 39864 36579 1261 9951 18383 10043 36041 46303 13368 42861 26049 28773 22602 14242 35402 36593 34473 36662 38679 12592 31109 13744 18481 45665 8743 19725 11059 42494 25034 18372 24638 10436 41841 48565 41110 8131 11485 25086 31668 9103 49548 46518 35774 36977 26634 30147 5722 915 24969 31338 45936 34421 884 27331 24085 20140 41013 3834 37016 13016 8241 13752 15560 35865 45489 27493 33237 31450 9986 23744 759 47759 35634 1249 30242 25244 
198
43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 43186 
15735 27646 33027 18780 22109 28910 16997 49400 27447 10686 19348 40835 42129 7704 14370 18504 227 6499 41337 4810 40445 46206 9638 32849 834 44030 7567 26645 25164 27479 35294 38213 29023 48767 38898 39217 741 40832 19536 38368 16071 48306 44662 32327 29790 46968 29698 44965 45559 29901 8170 37162 18902 23158 38864 28398 23653 8776 36377 45973 14874 22212 17881 39686 2241 10511 39187 47574 38827 26292 45695 47165 39270 44165 38428 12895 17228 25475 15293 35014 29748 29848 9578 4224 49871 38517 5322 28838 25489 19065 18963 32194 40890 42456 32586 30136 31884 34364 21776 19837 11122 5036 47174 40739 1102 34923 11633 45747 9909 43680 21077 21836 48613 30222 37531 17146 36757 37559 6874 9114 1800 43597 39057 33679 20374 48990 11422 1379 41328 13552 29332 19552 15496 18552 36975 16779 18076 33795 42470 42366 42922 33408 42178 46578 2763 7711 980 4284 20522 28334 25872 2077 33905 19752 13128 36077 18812 27455 6401 45258 9831 46093 23202 19587 37425 42818 38517 41600 47369 27438 29505 43913 38965 15462 40935 31425 17669 18964 33968 29903 23505 26468 42943 20254 22076 19349 20092 47057 22297 4165 13655 6703 40425 25212 16404 28671 31291 28622 
199
38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 38550 
12873 16360 13558 40049 11299 2888 42925 7180 14547 6025 34363 1009 45392 32607 37697 10858 26619 15626 27909 29610 18957 27398 43479 21080 27375 47856 13705 8610 30079 21899 6992 15512 31295 13469 18493 17810 28551 14675 2192 39163 22059 49530 34087 11907 8188 33724 46377 31973 17448 3888 37090 35401 29330 43991 39537 34691 4811 37284 49532 44602 32915 12324 28543 10225 48695 36053 41343 19364 33616 35616 42567 43671 17005 22168 43020 3471 13289 27467 8304 20931 47611 12550 10242 49410 48572 30524 8968 11789 40369 20077 45790 46341 37374 28588 36989 8216 29912 14273 6600 3875 19967 33830 26476 49176 35609 11368 27387 7946 3832 39224 29389 32401 43692 22468 36673 12846 30482 2697 7293 36359 27201 1593 30452 3002 1406 48471 3049 14850 11531 21803 6017 49134 24870 42948 3954 10708 31348 40419 33091 20019 10510 2289 15512 47610 41554 9712 41595 29362 28558 13819 24526 39292 25997 46517 30044 28903 26752 46665 47421 49634 31858 26057 24808 48768 21446 34207 15711 27089 7679 23329 48403 28169 15413 46866 13231 44693 12966 326 5015 33982 31064 9241 17607 16988 40909 18137 34957 21148 40014 35417 31522 25865 21241 7221 28139 26082 17311 47473 22781 
200
79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 79700 
9067 47257 36282 3252 6652 29321 7762 5324 33918 14435 24653 6600 19928 36795 22434 3359 70 25964 23513 19817 46947 6757 2614 20999 10222 6692 15286 16144 3083 28562 6007 29921 47596 30405 187 38158 14928 16012 49136 32208 18336 33047 36540 36387 36855 44004 43168 23040 37817 6376 46339 37573 14798 43980 22954 21429 26373 6707 15198 8503 41824 1569 23502 10196 14549 34499 26153 43208 27003 1714 41783 32855 5501 27457 21782 21592 44468 43866 10084 2294 21552 38480 41366 46738 8517 13372 17119 25508 39354 38113 4777 7206 5721 43836 25252 40609 27464 41019 37218 2966 21119 17918 44614 10492 42570 7067 37527 46472 11346 45464 44151 48473 11910 17200 23032 15668 35602 23655 2650 24741 20482 42355 7451 8202 215 35488 12569 12303 27650 20751 3333 20677 13402 41274 11699 47971 31206 35792 39245 40438 36026 14086 47707 17531 18129 26438 45032 45189 40832 42875 18779 16793 42475 9207 24852 35419 10421 49028 13657 9885 7808 23830 49240 9184 4143 7758 19180 3001 34103 983 10668 4979 37084 19816 7253 7521 18071 23536 7685 19191 47238 29752 29927 38354 17297 47073 8640 42739 29361 23882 178 21382 1315 14807 27651 1382 21887 5070 11827 46019 
199
83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 83390 
45852 38673 11420 4799 903 45311 9081 20691 49505 26094 21972 16308 39343 16344 48410 15247 43448 18600 14861 46755 11295 3720 30469 9337 41246 34343 18096 40679 29227 20133 11998 26376 32234 10922 48536 15457 18326 47734 43104 159 35357 19470 31594 34129 30263 47350 13502 39476 12924 40875 5417 28783 5239 18895 33178 28351 14119 19623 48754 6096 24800 44494 2751 25088 48001 6120 15109 48728 22139 33912 18975 29456 3261 7465 17938 7020 24187 37760 21300 25827 29224 46720 5450 59 10550 17752 7386 28323 7515 29453 4067 15644 30104 11607 40010 36359 4223 38944 25658 37546 5445 47131 20496 9453 34308 32372 18603 25181 49767 10172 38367 13790 10628 35975 7040 8380 25564 7020 15876 2887 26634 11637 47498 2614 40022 46521 15866 25210 4350 13043 32261 47638 15860 38630 11475 20368 40906 20923 40147 11884 8766 34095 40261 22198 26818 47454 19194 28050 14406 30705 5917 8126 45635 48419 14251 47212 46296 36667 36559 41152 14272 27560 40136 27469 29255 1693 44366 34793 47912 46759 21587 18378 11187 46290 18380 23305 49194 24811 43292 39633 47463 12766 2708 19024 41766 1718 5371 45147 14439 30508 41065 44716 1271 48565 15475 27233 37062 4795 48509 
197
88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 88104 
34970 3737 6487 21624 36211 7511 28620 45610 39393 36810 48759 28150 27404 12285 43254 34309 16118 31277 6523 44191 22506 44213 8478 18023 2790 11756 30685 24989 29239 3632 20990 32699 16543 35034 29292 28891 13475 16133 14610 33599 45780 45103 1571 43593 13010 24512 15042 37365 32503 9804 21776 23334 15136 24373 45442 14838 37901 20437 15432 35170 48696 25833 30892 8568 13338 40055 26136 33748 39657 42791 25974 34933 6596 48265 46340 40621 40705 10588 2011 2779 9776 39194 23474 1515 35109 41096 23268 27112 21392 22851 13825 12531 16601 5902 19750 1986 36677 35462 15237 38731 21246 3825 13680 12827 14201 28129 11968 16595 5615 33378 28236 45106 32003 13666 49465 38154 41309 39649 28091 39105 24553 37694 22700 13012 27364 24633 2986 20320 5784 6541 16537 38420 8523 37599 17675 49125 32646 39972 18038 42319 7664 2650 3650 22409 27039 22321 22083 41271 37832 20141 5367 22638 43530 9938 15452 6743 5068 425 35050 28515 38218 16837 49954 27420 25212 7336 3044 38389 37057 39095 18291 26152 23329 21968 39584 47543 10020 10749 30595 46870 21144 48192 44347 11973 30679 28629 20265 47967 11354 1652 34478 40453 25252 26801 30611 25477 37126 
197
23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 23523 
29126 36789 15835 32004 30997 49863 20976 43887 34451 5843 2977 41317 25749 21987 49025 32316 42517 17658 32371 40094 47198 4681 30216 7599 42664 30162 24485 48943 42160 1343 23295 32875 41571 12212 48494 111 4452 8392 24065 4350 20880 48859 38872 27174 13530 18406 149 46071 5434 47773 18773 14709 45129 16309 49605 45494 44317 29031 11868 29580 2018 40063 30254 4827 21209 15403 2921 31052 40179 39608 7637 26225 32311 16783 31304 42435 21556 32782 37739 40031 10815 36873 48231 45958 34048 35981 13183 27231 21784 28197 8800 34104 3611 9314 25343 18053 38596 31880 10304 24505 7693 24891 14306 33341 35847 38286 4611 17795 15889 38230 49253 9410 5079 21174 41627 9381 43797 45779 17379 21105 25914 27652 34133 29788 15397 36027 36001 34274 14265 17203 28108 44296 26812 136 42892 41569 10985 4757 14506 26041 23614 10957 19011 41793 31502 11749 5705 19500 25960 41924 10290 4733 33831 35020 45970 41817 23089 45737 5173 10422 11168 10009 12404 24570 49819 15380 26655 41974 29644 1049 32208 13404 1491 33431 36172 27575 31687 49630 2051 48741 43148 22029 25925 1223 29151 15828 36580 4698 27911 45022 32020 14455 29488 17558 589 29151 47949 
197
81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 81702 
47235 7729 20203 89 27879 232 30103 7917 33703 13370 1649 10606 19324 41909 24830 9504 11473 31774 44655 17396 21664 48484 7920 22435 30220 38808 23781 13298 19938 7831 45671 12410 15904 48823 35689 46224 37143 22133 13038 36743 33404 25320 45686 17679 43205 30677 13612 480 23158 15077 26340 16287 47035 23641 16524 13729 36716 40976 46093 48798 2049 13473 48511 5378 8669 7081 42397 43150 32702 5183 33899 10069 10586 20792 38555 36480 32111 3005 11553 855 20911 38623 15155 29926 23058 4761 43791 47939 1274 15262 7990 44430 49390 47473 6740 13745 24278 18998 15600 7699 30496 41767 36566 24364 49520 1454 48983 27295 25088 13663 15291 22270 13806 14766 6052 19938 24304 29019 21882 39460 48145 10914 30597 20480 35886 4245 27047 37433 39733 44508 47828 496 15138 21451 24250 30606 9616 345 31558 30038 13992 24994 4257 38808 14007 23068 32907 10061 32967 14633 28171 30667 45366 33717 22311 28609 15418 33048 558 48104 38246 34219 1935 25957 10172 5549 184 43913 47192 48785 37775 181 33053 21418 40626 47995 43008 45906 48273 28874 1567 12893 29286 18724 29602 6384 37025 29800 22915 28297 38776 6727 43704 33951 32871 39889 47096 
