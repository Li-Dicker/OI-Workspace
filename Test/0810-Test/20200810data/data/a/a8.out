Case #1: 12410466
Case #2: 20893635
Case #3: 13415831
Case #4: 13308248
Case #5: 12243712
Case #6: 19793982
Case #7: 20661428
Case #8: 21835403
Case #9: 8968396
Case #10: 20230883
